src/Microsoft.DotNet.Cli.Utils/FrameworkDependencyFile.cs

```diff
@@ -45,6 +46,36 @@ public string GetNetStandardLibraryVersion()
                 ?.Version;
         }
 
+        public bool TryGetMostFitRuntimeIdentifier(IEnumerable<string> candidateRuntimeIdentifiers, out string mostFitRuntimeIdentifier)
+        {
+            mostFitRuntimeIdentifier = null;
+
+            if (!SupportsCurrentRuntime())
+            {
+                return false;
+            }
+
+            RuntimeFallbacks runtimeFallbacks
+                = DependencyContext.RuntimeGraph.Where(g => g.Runtime == RuntimeEnvironment.GetRuntimeIdentifier()).Single();
+
+            if (candidateRuntimeIdentifiers.Contains(runtimeFallbacks.Runtime))
```

Comment:
Are RIDs supposed to be case-insensitive?

Reply:
NuGet uses IgnoreCase

https://github.com/NuGet/NuGet.Client/blob/a4f56680763408cb0433e7ffa59205444cd8e77e/src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFile.cs#L104

I don't think the host does though...

https://github.com/dotnet/core-setup/blob/4b628f43fc5989c8a3923cb074b905dbc458f11c/src/corehost/cli/deps_format.h#L25-L28

Reply:
We should follow nuget here since we're picking a folder from a package to use.